precompile/modules/initia_stdlib/sources/multisig_v2.move

```diff
@@ -162,6 +159,13 @@ module initia_std::multisig_v2 {
         tiers: Option<vector<Tier>>
     }
 
+    // view functions
+    public fun is_exist(creator_addr: address, name: &String): bool {
```

Comment:
If it is view functions, then we need to put `#[view]`

Reply:
oops, done